collections/_article/dev-snapshot-godot-4-4-beta-1.md

```diff
@@ -0,0 +1,291 @@
+
+#### Add support for AgX tonemapping
+AgX is the cool, new kid on the block when it comes to tonemapping algorithms. Last year Blender made waves by replacing their previous "filmic" tonemapper with AgX. We have implemented a version of AgX that closely matches Blender's while being not quite as high quality, but much more suitable for real time. AgX is a stylized tonemapper intended to give a filmic quality to images that handles very bright scenes much better than our other tonemappers.
+
+![ACES tonemapping left AgX right](/storage/blog/godot-4-4-beta-1/tonemap.webp)
+
+#### Add transparency support for LightmapGI 
+Currently when baking lightmaps users have to choose between transparent objects casting shadows as if they were fully opaque, or not casting shadows at all. This has been a major limitation in both the quality of lightmap baking and the ergonomics of the lightmap baking workflow. 
+
+In [GH-99538](https://github.com/godotengine/godot/pull/99538), Hendrik Brucker added support for baking with transparent objects, including having tinted shadows (something that dynamic shadows don't support yet). Since this is brand new in Beta 1, please test it carefully and report any issues.
+
+![Lightmap shaders with color](/storage/blog/godot-4-4-beta-1/lightmap-color.webp)
+
+### Breaking Changes
+We try to minimize breaking changes, but sometimes they are necessary in order to fix high priority issues. Where we do break compatibility, we do our best to make sure that the changes are minimal and require few changes in user projects.
+
+You can find a list of such issues by filtering the merged PRs in the 4.3 milestone with the [`breaks compat` label](https://github.com/godotengine/godot/issues?q=milestone%3A4.4%20is%3Amerged%20label%3A%22breaks%20compat%22). Here’s a selection of some which are worth being aware of:
+
+- Universal UID support. This will create several `.uid` files for resources that lacked this metadata. **These should be added to version control**, much like `.import` files. ([GH-97352](https://github.com/godotengine/godot/pull/97352))
+- Floats converted to strings now display as a decimal by default, even for whole numbers. This makes printed types more explicit, which can be used to catch bugs that were previously invisible. That is: `print(1.0)` will now print `1.0` instead of `1`. ([GH-47502](https://github.com/godotengine/godot/pull/47502))
+- Changed `KEY_MODIFIER_MASK` to correct value. Unlikely to directly affect any projects, as the previous value was simply wrong. ([GH-98441](https://github.com/godotengine/godot/pull/98441))
+- Control's `offset_*` get/set type changed from `int` to `float`. This matches the behavior described in the documentation. ([GH-98443](https://github.com/godotengine/godot/pull/98443))
+- `OpenXR` action maps opened in Godot 4.4 aren't compatible with earlier versions. ([GH-TODO](https://github.com/godotengine/godot/pull/TODO))
```

Comment:
@dsnopek Could you point me to the right PR for this?

Reply:
Sure! https://github.com/godotengine/godot/pull/98163